### ISSUE TITLE
Fix first run permissions bug!

### ISSUE THREAD

**Issue by MasterTailer** [MasterTailer/DIGITAiL#72]

**Comment by MasterTailer** [comment]
Which means that it cant be run the first screen. App needs to be closed and reopened.

**Comment by gilmanov-ildar** [comment]
The source of the bug is into the `TailActivity::onPause()` method, it seems we should not call `finish()` method when we ask permissions.

I commented that code at the branch `call-handlers`, so now it works well.
@leinir, did you really need the `finish()` call into the `onPause()` method?

**Comment by gilmanov-ildar** [comment]
From the `Activity` documentation (https://developer.android.com/reference/android/app/Activity#finish()):

```
public void finish ()

Call this when your activity is done and should be closed. The ActivityResult is propagated back to whoever launched you via onActivityResult().
```

**Comment by leinir** [comment]
> The source of the bug is into the `TailActivity::onPause()` method, it seems we should not call `finish()` method when we ask permissions.
> 
> I commented that code at the branch `call-handlers`, so now it works well.
> @leinir, did you really need the `finish()` call into the `onPause()` method?

Good point indeed! The two bits of code conflict entirely, and it makes a whole lot of sense that it would fail the way it does. Thanks for spotting that :)

The reason the finish call is in the onPause() method is to work around a bug in Qt which is due to be fixed in 5.14: https://bugreports.qt.io/browse/QTBUG-70772
